homeassistant/components/netgear/device_tracker.py

```diff
@@ -76,6 +76,9 @@ def update(self) -> None:
         if not self._active:
             self._attrs = {}
 
+        if self.entity_id:
```

Comment:
This should not be necessary since you only attach in `async_added_to_hass`, where we have an entity_id.